presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -133,6 +133,8 @@ String
 Date and Time
 -------------
 
+Also see :doc:`/language/time`
```

Comment:
We use "see also" in the docs. I picked that up from PostgreSQL, but it seems to be the proper way of writing citations.